internal/karma/karma.conf.js

```diff
@@ -4,6 +4,86 @@ const path = require('path');
 const fs = require('fs');
 const tmp = require('tmp');
 
+const WORKSPACE_NAME = 'TMPL_workspace_name';
+
+// On Windows, runfiles will not be in the runfiles folder but instead
+// there is a MANIFEST file which maps the runfiles for the test
+// to their location on disk. Bazel provides a RUNFILE_MANIFEST_FILE environment
+// variable which is set the location of the MANIFEST file during test execution.
+// If a MANIFEST file is found, we remap the test files to their
+// location on disk using the MANIFEST file.
+let manifest = null;
+if (process.env.RUNFILES_MANIFEST_ONLY === '1' &&
```

Comment:
we should throw an error if RUNFILES_MANIFEST_ONLY=1 and the file doesn't exist ?